packages/mobile/src/fiatExchanges/Ramp.tsx

```diff
@@ -0,0 +1,86 @@
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useEffect, useRef } from 'react'
+import { BackHandler, StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import WebView, { WebViewRef } from 'src/components/WebView'
+import { VALORA_LOGO_URL } from 'src/config'
+import { CURRENCY_ENUM } from 'src/geth/consts'
+import config from 'src/geth/networkConfig'
+import i18n from 'src/i18n'
+import { emptyHeader } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarTextButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+const RAMP_URI = config.rampWidgetUrl
+
+const navigateHome = () => navigate(Screens.WalletHome)
+
+export const rampOptions = () => {
+  const navigateToFiatExchange = () => navigate(Screens.FiatExchange)
+  return {
+    ...emptyHeader,
+    headerTitle: (RAMP_URI.match(/(?!(w+)\.)\w*(?:\w+\.)+\w+/) || [])[0],
+    headerLeft: () => (
+      <TopBarTextButton title={i18n.t('global:done')} onPress={navigateToFiatExchange} />
+    ),
+  }
+}
+
+type RouteProps = StackScreenProps<StackParamList, Screens.Ramp>
+type Props = RouteProps
+
+function FiatExchangeWeb({ route }: Props) {
```

Comment:
Lets call the function the same as the file for clarity. Also, I know we haven't been consistent about this but I find it helps if screens have the `Screen` suffix so this file/function could be called `RampScreen`